### ISSUE TITLE
Is v4 using cryptoRNG as defined in documentation

### ISSUE THREAD

**Issue by jonasfj** [Daegalus/dart-uuid#32]
Documentation comments seems to suggest that `uuid.v4()` uses an AES based random source:
https://github.com/Daegalus/dart-uuid/blob/15b232b775d6e60443bd9b67bc0762fc9a0b08ef/lib/uuid.dart#L202-L203

But the code suggest that it doesn't:
https://github.com/Daegalus/dart-uuid/blob/15b232b775d6e60443bd9b67bc0762fc9a0b08ef/lib/uuid.dart#L225-L227
(or am I reading it wrong?)

Note. people are likely to rely on v4 security properties for things like generating a request nonce, etc..

**Comment by Daegalus** [comment]
Sorry, that comment is outdated, I had to default to non-Crypto a while back to reduce sizes for client code.

the Readme is updated and states this:
```
Cryptographically strong random number generation on all platforms
- Defaults to non-crypto generator, see UuidUtil for cryptoRNG
```

So you will have to specify the cryptoRNG function in your options that you pass into the v4 function.

**Comment by jonasfj** [comment]
I haven't check what platforms it's available on, but have you considered:
https://api.dartlang.org/stable/2.1.0/dart-math/Random/Random.secure.html

Anyways, it's probably good to write this in bold in case people depend on the package.

**Comment by Daegalus** [comment]
I wasn't aware of that. I will add that to my v2 changes. When I originally wrote this, Random.secure didn't exist.

**Comment by Daegalus** [comment]
Fixed it in v2.0.0-rc1 which is out, and i made sure to fix the comments and the Readme to more prominently say cryptoRNG is not the default.
